content/news/2024-08-21-hosting-shiny-apps-on-EU/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: 'Hosting R/Shiny applications on Galaxy (and other frameworks)'
+date: '2024-08-21'
+tease: "Thanks to the outcome of a small Galaxy/Bioconductor hackathon earlier this year, it has become much easier to deploy a Shiny phyloseq App as an interactive tool (IT) on usegalaxy.eu"
```

Comment:
Should this be more general (i.e. not specifically for phyloseq)? E.g. 
"Thanks to the outcome of a small Galaxy/Bioconductor hackathon earlier this year, it has become much easier to deploy Shiny applications as interactive tools (ITs) on usegalaxy.eu"

Reply:
looks like this is a typo from my side....I will fix it

Reply:
ah, the github wizard has already done it - Thanks